File: Dominate/app/src/main/java/com/jeff/dominate/fragments/MeFragment.java
package com.jeff.dominate.fragments;

import android.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.jeff.dominate.R;

/**
 * author : Jeff  5899859876@qq.com
 * CSDN ： https://blog.csdn.net/Jeff_YaoJie
 * Github: https://github.com/Jay-YaoJie
 * Created :  2018-12-13.
 * description ： 我的页面
 */
public class MeFragment extends Fragment {


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        View view = inflater.inflate(R.layout.fragment_me,
                null);

        return view;
    }
}

File: Dominate/app/src/main/kotlin/me/meFragment.kt
package me

import bases.BaseFragment
import com.jeff.dominate.R

/**
 * author : Jeff  5899859876@qq.com
 * CSDN ： https://blog.csdn.net/Jeff_YaoJie
 * Github: https://github.com/Jay-YaoJie
 * Created :  2018-12-21.
 * description ：meFragment 我的页面
 */
class MeFragment : BaseFragment<MeFragmentDB>() {
    override fun getContentViewId(): Int= R.layout.fragment_me

    override fun initViews() {
           }

}